prisma/schema.prisma

```diff
@@ -33,6 +33,7 @@ model User {
   createdAt         DateTime  @default(now())
   updatedAt         DateTime  @updatedAt()
   deletedAt         DateTime?
+  address Address? 
```

Comment:
A relação do Address p/ user é One-to-many, possibilitando o user ter 1 ou varios endereços, então deve definir o campo como: addresses Address[], e ñ precisa tornar o campo opcional.